pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -24,6 +24,9 @@ import (
 	"strings"
 
 	"github.com/go-logr/logr"
+	litmuschaosv1alpha1 "github.com/litmuschaos/chaos-operator/pkg/apis/litmuschaos/v1alpha1"
+	"github.com/litmuschaos/chaos-operator/pkg/controller/resource"
+	chaosTypes "github.com/litmuschaos/chaos-operator/pkg/controller/types"
```

Comment:
These changes are not required!

Reply:
removing them results in errors
![image](https://user-images.githubusercontent.com/22005947/71587409-90c73900-2b43-11ea-835e-7a558a8e5e5d.png)

Reply:
You should not remove it should move to older place, Where it was earlier check in the PR diff

Reply:
ok done

Reply:
Done @chandankumar4